### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by BlackBoxAstro** [beltoforion/stellarium-to-video#13] [General]
Hello, 

I am just posting to check/confirm you've released this code to the public domain? 

You have written public domain in the script section of the code but I just want to confirm that this was intended to apply to the whole code. 

I have made some significant changes and published them to a new repo (attributing to you of course) instead of forking as there were quite a few changes. Link is here: https://github.com/BlackBoxAstro/Stellarium-Video-Maker 

Thanks,
Sean

**Comment by beltoforion** [comment]
You can use it in that manner. I will probably update the archive and put it under the BSD License explicitly in the future. This will not change anything for you.

**Reply by BlackBoxAstro** [reply]
Thank you :)
